src/Uno.UI/UI/Xaml/DependencyPropertyDetailsCollection.cs

```diff
@@ -106,6 +106,8 @@ public void Dispose()
 			}
 
 			ReturnEntriesAndOffsetsToPools();
+
+			_entries = null!;
```

Comment:
Maybe we should change the `_entries` declaration to allow nulls here?

Reply:
@carldebilly Probably not. Being null isn't really the "valid" state. It happens only after disposal so I don't think we should annotate it as null.